app/code/Magento/InventorySales/Model/SalesEventToArrayConverterInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventorySales\Model;
+
+use Magento\InventorySalesApi\Api\Data\SalesEventInterface;
+
+/**
+ * @api
+ */
+interface SalesEventToArrayConverterInterface
```

Comment:
Should we have this as an interface but not a private model?

Reply:
Hi @naydav! My idea was to provide an extension point for developers because there could be a need to add more meta data.

Reply:
agree with @naydav , made according fix.
Provided more info to @vadimjustus about reasoning behind this in Slack